yara/YaraLexer.g4

```diff
@@ -0,0 +1,214 @@
+/*
+YARA grammar.
+The MIT License (MIT).
+
+Copyright (c) 2022, Michał Lorek.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+lexer grammar YaraLexer;
+
+options { caseInsensitive = false; }
+
+ALL                       : 'all';
+AND                       : 'and';
+ANY                       : 'any';
+//ASCII                     : 'ascii';
+AT                        : 'at';
+//BASE64                    : 'base64';
+//BASE64WIDE                : 'base64wide';
```

Comment:
What's the point of such commented out tokens? It would be more clear to have them removed.